src/output/render-manager.cpp

```diff
@@ -348,6 +353,18 @@ class wf::render_manager::impl
         on_frame.connect(&output_damage->damage_manager->events.frame);
 
         init_default_streams();
+
+        background_color_opt_changed = [=] ()
+        {
+            auto color = background_color_opt->as_color();
+            update_background_color(color);
+        };
+
+        auto section = wf::get_core().config->get_section("core");
+        background_color_opt = section->get_option("background_color", "0 0 0 1");
+        background_color_opt->add_updated_handler(&background_color_opt_changed);
```

Comment:
One more thing: when the render manager is destroyed, we need to disconnect the updated handler.